### ISSUE TITLE
Exchange Online PowerShell V2 Module requires .Net Framework 4.7.1 or higher.

### ISSUE THREAD

**Issue by komiyamo** [MicrosoftDocs/office-docs-powershell#9810]
This document states that Windows PowerShell 5.1 or higher is required as a condition for a Windows environment, that needs .Net Framework 4.5 or higher as a prerequisite.

However, if .Net Framework 4.7.1 or higher is not installed, Connect-ExchangeOnline or Connect-IPPSSession command is failed because "System.Runtime.InteropServices.OSPlatform" cannot be loaded.
This is because the OSPlatform structure is introduced in .Net Framework 4.7.1 and above.

Therefore, it would be better to state in the documentation that .Net Framework 4.7.1 or higher is required as a prerequisite for using Exchange Online PowerShell V2 Module for windows environment.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: de2836a0-0af3-c147-2d4a-78220ac4e0f1
* Version Independent ID: 2cde9735-6281-bb9f-bf07-b30b2f6c7f37
* Content: [About the Exchange Online PowerShell V2 module](https://docs.microsoft.com/en-us/powershell/exchange/exchange-online-powershell-v2?view=exchange-ps&source=docs)
* Content Source: [exchange/docs-conceptual/exchange-online-powershell-v2.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/docs-conceptual/exchange-online-powershell-v2.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@komiyamo, which version of Windows are we talking about?

**Comment by chrisda** [comment]
And which version of the module are we talking about: v2.0.5 or one of the Preview versions of v2.0.6?

**Comment by komiyamo** [comment]
This can be an issue on versions of Windows that do not have .Net Framework 4.7.1 or higher installed by default.
Note: Some versions of Windows 10 include .Net Framework 4.7.1 or higher by default, so this may not be an issue.
https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies

Regarding the PowerShell module, we have confirmed this error behavior in v2.0.5.

**Comment by chrisda** [comment]
@komiyamo, OK. I reworked the Windows prerequisites section to make v2.0.5 and .NET 4.7.1 prominent, and deemphasized the bulleted list of OS'es.

The topic already states that Windows 7 supports only v2.0.3 and earlier of the module, which is apparently regardless of its support for .NET 4.7.1. IT will be interesting to see if other incompatibilities fall out of this.

Thanks for the feedback.

Closing
